common/common.scss

```diff
@@ -1,30 +1,17 @@
-.footer {
-    background-color: #4f4f4f;
-    color: white;
-    padding: 20px;
-    display: flex;
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: url($roboto) format('woff2')
```

Comment:
Il y a une chance pour que [ça fail dans IE 11](https://caniuse.com/woff2) en ne mettant que le support de woff2. Ça n'est pas un problème si vous avez peu de visiteurs sous ce navigateur mais il faut peut-être vérifier ?

Reply:
Bien vu. Je n'ai pas le détail par version d'IE, mais 5% des utilisateurs ont [Trident](https://fr.wikipedia.org/wiki/Trident_(moteur_de_rendu)#Versions) comme moteur de rendu, donc des versions <= IE 11. Je vais voir pour ajouter les fallback.

Reply:
Je suis finalement passé par les font hébergées sur le CDN google. De base c’est un peu pénible d’ajouter beaucoup d’assets dans un theme, même [les gens de discourse se contentent du woff2](https://meta.discourse.org/t/create-and-share-a-font-theme-component/62462) dans la doc sur les fonts :(.
Si on a des problèmes de performances on pourra toujours corriger mais je me dis qu’on peut commencer simple.